modules/core/TextureMarble.cpp

```diff
@@ -0,0 +1,62 @@
+//  Created by Mahmoud El Bergui on 21.04.22.
+//
+
+#include "TextureMarble.h"
+#include "PerlinNoise.h"
+
+namespace {
+
+     Vec3f marbleMap(float value){
+               
+          Vec3f white = Vec3f(1,1,1);
+          Vec3f blue = Vec3f(0.6 , 0.53 , 0.467);
+          Vec3f color = (Vec3f) ((white - blue) * value + blue);
+               
+               return color;
+     }
+     
+     double smoothstep(float val){
```

Comment:
This method is never used

Reply:
As I was building the Marble texture, I found that it would be useful to have these functions. One can just use them without having to write the code for them. I will send you a picture showcasing the difference between the two results with smooth step and without.